serverless/packages/functions/src/trainspace/create_image_trainspace.ts

```diff
@@ -0,0 +1,49 @@
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+import parseJwt from "@dlp-sst-app/core/parseJwt";
+import { v4 as uuidv4 } from 'uuid';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient, PutCommand } from '@aws-sdk/lib-dynamodb';
+import { create_trainspace } from "../dbutils/put_trainspace";
+
+export const handler: APIGatewayProxyHandlerV2 = async (event) => {
+    if (event) {
+        const uid: string = parseJwt(event.headers.authorization ?? "")["user_id"];
+        const eventBody = JSON.parse(event.body? event.body : "");
+
+        const trainspaceId = uuidv4();
+        const putCommandInput = create_trainspace(trainspaceId, uid, "IMAGE", eventBody['dataset_data'], eventBody['name'], eventBody['parameters_data'], eventBody['review_data']['notification_email']);

```

Comment:
creating the trainspace object shouldn't be generalized to a function, since the attributes for each trainspace can vary between trainspace types (image, tabular, etc)

Reply:
whats stopping us from having separate trainspace creation functions? that way each trainspace we create has a predictable structure with some differences? rather than hard coding all props of trainspace down at sst? @danielwu